src/function/aggregate/distributive/entropy.cpp

```diff
@@ -9,8 +9,23 @@ namespace duckdb {
 
 template <class T>
 struct EntropyState {
+	using DistinctMap = unordered_map<T, idx_t>;
+
 	idx_t count;
-	unordered_map<T, idx_t> *distinct;
+	DistinctMap *distinct;
+
+	// NOLINTNEXTLINE(bugprone-unhandled-self-assignment)
+	EntropyState &operator=(const EntropyState &other) {
```

Comment:
Perhaps use an explicit `Copy` function, so it is clear we are making an (expensive) copy and we can avoid the linter exception?

Reply:
It's always the same with those darn computers isn't it :)